packages/components/src/common/postMixin.js

```diff
@@ -106,6 +110,10 @@ export default {
   },
 
   computed: {
+    shouldOpenNewTab() {
+      return this.openNewTab ? "_blank" : "_self";
```

Comment:
Please make sure to run `yarn lint --fix`, e.g we do not use double quotes.
I need to fix the CI to throw error on linting issues 😅

Reply:
Oh sorry, I forgot. Lol.

Reply:
Hi bro, I tried running the command and it was not changed, though I already modified it manually :grin:

You might get interested on `Prettier`. It automatically formats the code on save (default) based on configuration also and has a nice VS Code extension too!